runtime/src/lib.rs

```diff
@@ -552,6 +560,7 @@ impl_runtime_apis! {
             add_benchmark!(params, batches, frame_system, SystemBench::<Runtime>);
             add_benchmark!(params, batches, pallet_balances, Balances);
             add_benchmark!(params, batches, pallet_timestamp, Timestamp);
+            add_benchmark!(params, batches, pallet_utility, Utility);
```

Comment:
I wonder if this is really needed. How can we benchmark the utility pallet on our reference machine, without having to keep a copy of the pallet and update it manually or even fork the whole substrate repository and benchmark all necessary pallets regularly? AFAIK, we have to replace the `weights.rs` info file that contains the `WeightInfo` struct, and I don't see how this is possible without maintaining a copy of the pallet or substrate. In that case, is it really necessary to include the benchmark and bloat the runtime (that's built with *--feature runtime-benchmarks*) with code that never will be executed?

Reply:
Maybe the easiest way to handle this issue is to make our standard hardware aligned with Polkadot's so that we can use the Substrate benchmarks and weights. Otherwise it's inescapable to re-benchmark them with our standard machine afaict.